src/settings/OpenExceptionalForm/ExceptionalPeriodEditor.js

```diff
@@ -203,13 +203,14 @@ class ExceptionalPeriodEditor extends React.Component {
         <Row>
           <Col>
             <div data-test-service-points>
-              <div data-test-service-points-title>
+              <Label required data-test-service-points-label>
```

Comment:
As you already change this line, could you please split properties into different lines. Also could you please apply this approach for all lines abowe and below, which you change and where number of properties is more than one.
Thanks!